### DISCUSSION TITLE
How to fix vulnerability detected by OWASP Zap

### DISCUSSION THREAD

**Discussion by chojisan** [zitadel/zitadel#8969] [Q&A]
I attached the report generated from OWASP ZAP, this also affects the latest version of zitadel. Detected are more on CSP headers.

Edit: deleted the file by @fforootd

**Comment by mffap** [comment]
Please check the guidelines on https://zitadel.com/docs/security.txt for more info, accepted scope, and process to handle findings.

I haven't checked your zip file. We do have a responsible disclosure policy (linked above) - that should explain how we handle vulnerability reports and how you should report such vulnerabilities. Thanks.

**Reply by chojisan** [reply]
ok, sorry. I'll check the guidelines. thank you
